apis/python/src/tiledbsoma/dense_nd_array.py

```diff
@@ -125,6 +125,9 @@ def shape(self) -> NTuple:
             return cast(NTuple, A.schema.domain.shape)
 
     def reshape(self, shape: NTuple) -> None:
+        """
```

Comment:
Optional/nit:  please add maturity tags whenever you update docstrings (even though `experimental` is the default, we should still eventually add them).